klarna-payments-for-woocommerce.php

```diff
@@ -135,6 +135,23 @@ public function init() {
 				return;
 			}
 
+			if ( ! class_exists( 'WC_Payment_Gateway' ) ) {
+				add_action(
+					'admin_notices',
+					function () {
```

Comment:
Avoid using anonymous functions for actions and filters. Please break this out into a separate method or function instead. Should be following WordPress coding standards unless we have a good reason for it.
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#closures-anonymous-functions